### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by khk-optesis** [odoo/odoo#28955]
Impacted versions: Odoo 12

Steps to reproduce: payslip batches

Current behavior: Error shows

Expected behavior:

Video/Screenshot link (optional):

<img width="737" alt="error" src="https://user-images.githubusercontent.com/44680721/48905205-cdad0080-ee58-11e8-9fe1-0f92cfa0ef9a.PNG">

**Comment by khk-optesis** [comment]
i reproduce the in runbot and that work but at local i install hr_payroll_account and when i try to generate payslip batches i have this error

**Comment by Yenthe666** [comment]
Have you updated your Odoo source code to the latest version? Do you also get the error in English?

**Comment by khk-optesis** [comment]
I have no mistake in English, I suppose the mistake is the translation.

**Comment by Yenthe666** [comment]
@khk-optesis yes in that case there is a translation term that is incorrect. You should find and fit it at https://transifex.com/odoo/

**Comment by Yenthe666** [comment]
You're welcome, best of luck!
